sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -189,12 +195,14 @@ class SdkSynchronizer private constructor(
     }
 
     private val _status = MutableStateFlow<Synchronizer.Status>(DISCONNECTED)
+    private val _exchangeRateUsd = MutableStateFlow<Pair<BigDecimal, Instant>?>(null)
```

Comment:
The same comment as above for the `Pair` usage.

Reply:
Fine with me (this type is entirely Kotlin-side, I just did what was sufficient to get this working).